test/e2e/framework/framework_constants.go

```diff
@@ -80,11 +80,13 @@ const (
 	CLIE2ETestEnvironment = "TANZU_CLI_E2E_TEST_ENVIRONMENT"
 
 	// General constants
-	True            = "true"
-	Installed       = "installed"
-	UpdateAvailable = "update available"
-	NotInstalled    = "not installed"
-	JSONOtuput      = "-o json"
+	True             = "true"
+	Installed        = "installed"
+	UpdateAvailable  = "update available"
+	NotInstalled     = "not installed"
+	RecommendInstall = "install recommended"
+	RecommendUpdate  = "update recommended"
```

Comment:
Nit: it could just be me, but seems like I can read "update recommended" two ways (similarly for L87)
- "an update is recommended for this plugin" or
- "please update to the recommended version of this plugin"

The first is a state of the plugin, and the correct interpretation
The second is an action, and does not make sense in the STATUS column
unfortunately the first word being a verb has led me to interpret the second way a few times already.

moreover "installed recommended" makes it harder to visually differentiate from the list output the list of plugins that are "installed" vs the ones that are not.

 Some suggestions for alternatives I have are:
"needs update/install"
"require update/install"
"update needed"/"not installed"

I forgot the rationale of moving away from "not installed", but that still felt a rather obvious option imo.

Reply:
We had async discussion and decided to go with `update needed"/"not installed`